.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+
+### Description
+Please provide a clear and concise description of your changes. Explain **what** this pull request accomplishes and *
+*why** it is needed.
+
+---
+
+### Related Issue
+
+If this pull request addresses an existing issue, please link to it below. Use the `Fixes #<issue-number>` syntax to
+close the issue automatically when the pull request is merged.
+
+Example:
+
+- Fixes: #1234
+
+---
+
+### Contributor License Agreement (CLA)
+
+By contributing to this project, you agree to the terms of
+the [Canonical Contributor License Agreement (CLA)](https://ubuntu.com/legal/contributors).  
+If you have not already signed the CLA, please do so [here](https://ubuntu.com/legal/contributors).
```

Comment:
```suggestion
If you have not already signed the CLA, [please do so here](https://ubuntu.com/legal/contributors).
```

Reply:
For accessibility, it's better to include a slightly longer text for links so it provides a larger area for people using magnifiers to click on